x-pack/platform/plugins/shared/stack_connectors/server/connector_types/openai/openai.ts

```diff
@@ -59,13 +59,21 @@ export class OpenAIConnector extends SubActionConnector<Config, Secrets> {
   private provider;
   private key;
   private openAI;
+  private headers;
 
   constructor(params: ServiceParams<Config, Secrets>) {
     super(params);
 
     this.url = this.config.apiUrl;
     this.provider = this.config.apiProvider;
     this.key = this.secrets.apiKey;
+    this.headers = {
```

Comment:
I think perhaps for more complex schema changes, we might want this wrapped with the experimental flag check, but this seems pretty safe since it's internally just adding more bits to the existing headers.  In theory though, someone using the API vs UX would be able to set these, without having the flag set.  

Seems like the risk here is very minor though, for their to be breakage without using the flag, someone would have to create the connector WHILE an upgrade was happening.  Or after an upgrade, and then a roll back (which we've never done).